mutiny-wasm/src/lib.rs

```diff
@@ -1756,6 +1756,36 @@ impl MutinyWallet {
         Ok(())
     }
 
+    /// Gets the list of npubs we're following
+    pub async fn get_follow_list(
+        &self,
+    ) -> Result<JsValue /* Vec<nostr::PublicKey> */, MutinyJsError> {
+        let list = self.inner.nostr.get_follow_list()?;
+        let npubs: Vec<String> = list
+            .into_iter()
+            .map(|n| n.to_bech32().expect("bech32"))
+            .collect();
+        Ok(JsValue::from_serde(&npubs)?)
+    }
+
+    /// Follows the npub on nostr if we're not already following
+    ///
+    /// Returns true if we're now following, false if we were already following
+    pub async fn follow_npub(&self, npub: String) -> Result<bool, MutinyJsError> {
```

Comment:
I don't really understand the purpose of these bools. Doesn't seem useful, just confusing.

Reply:
Felt it was better to return something rather than `()`

Reply:
I don't think it matters. It either follows them or errors. Nothing else really matters.